app/src/main/groovy/ua/net/nlp/api/BatchController.groovy

```diff
@@ -29,8 +29,19 @@ class BatchController extends BaseController {
     @Autowired
     BatchService batchService
 
+    def validateRequest(BatchRequest request) {
```

Comment:
Could you also please comment on this one?

Reply:
```
request.texts.each { 
if( ! it ) ... 
if( it.length() > .... ) ...
}
```